sky/clouds/azure.py

```diff
@@ -39,9 +39,9 @@
 _DEFAULT_AZURE_UBUNTU_2004_IMAGE_GB = 150
 _DEFAULT_SKYPILOT_IMAGE_GB = 30
 
-_DEFAULT_CPU_IMAGE_ID = 'skypilot:gpu-ubuntu-2204'
-_DEFAULT_GPU_IMAGE_ID = 'skypilot:gpu-ubuntu-2204'
-_DEFAULT_V1_IMAGE_ID = 'skypilot:v1-ubuntu-2004'
+_DEFAULT_CPU_IMAGE_ID = 'skypilot:custom-cpu-ubuntu-v2'
+_DEFAULT_GPU_IMAGE_ID = 'skypilot:custom-gpu-ubuntu-v2'
+_DEFAULT_V1_IMAGE_ID = 'skypilot:custom-gpu-ubuntu-v1'
```

Comment:
Should we add the automatic refetch images catalog logic for azure as well?

Reply:
What kind of automatic refetch images? The azure images.csv actually doesn't need to be updated at all

Reply:
Something like this? https://github.com/skypilot-org/skypilot/pull/4127

Reply:
Oh you mean local images.csv, got it